routes/reviews.js

```diff
@@ -4,6 +4,21 @@ const mongoose = require('mongoose')
 const Review = require('../models/review')
 
 /* GET Reviews listing. */
+router.get('/course/:courseId', function (req, res, next) {
```

Comment:
this was moved up?

Reply:
Yeah i moved it up because it's more specific